flycheck.el

```diff
@@ -4335,6 +4349,34 @@ universal prefix arg, and only the id with normal prefix arg."
       (seq-do #'kill-new (reverse messages))
       (message (string-join messages "\n")))))
 
+(defun flycheck-explain-error-at-point ()
+  "Display an explanation for the first error at point."
+  (interactive)
+  (let* ((errors-at-point (flycheck-overlay-errors-at (point)))
+         (first-error (car errors-at-point))
+         (checker (and first-error (flycheck-error-checker first-error)))
+         (explainer (and first-error (flycheck-checker-get checker
+                                                           'error-explainer)))
+         (explanation (and explainer (funcall explainer first-error))))
+    (cond
+     ((null first-error) (message "No errors at point"))
+     ((null explainer) (message (format "No explainer for %S checker" checker)))
+     ((null explanation) (message "No explanation"))
+     (t (flycheck-display-error-explanation explanation)))))
+
+(defun flycheck-display-error-explanation (explanation)
+  "Display the given error explanation in a help buffer."
+  (message explanation)
+  (with-help-window (get-buffer-create "*flycheck-explain-error*")
+    (princ explanation)))
```

Comment:
`message` was for debugging while trying to find the right way to write in the help window.